platform/metrics.go

```diff
@@ -0,0 +1,97 @@
+package platform
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+
+	"github.com/weaveworks/flux"
+)
+
+type Metrics struct {
+	AllServicesDuration  metrics.Histogram
+	SomeServicesDuration metrics.Histogram
+	RegradeDuration      metrics.Histogram
+	PingDuration         metrics.Histogram
+}
+
+func NewMetrics() Metrics {
+	return Metrics{
+		AllServicesDuration: prometheus.NewSummaryFrom(stdprometheus.SummaryOpts{
```

Comment:
Do we definitely want summaries here? As I understand it, they cannot be aggregated when querying.